src/lib/Hydra/Schema/Jobsets.pm

```diff
@@ -412,12 +412,30 @@ __PACKAGE__->add_column(
 
 my %hint = (
     columns => [
+        "errortime",
+        "lastcheckedtime",
+        "triggertime",
         "enabled",
```

Comment:
Should enabled be a boolean column?

Reply:
No, because it has 4 states: enabled, one-shot, one-at-a-time, and disabled.

Reply:
:upside_down_face: cool